app/components/Feed/index.js

```diff
@@ -32,7 +33,7 @@ const Feed = ({ items }: Props) => (
       })
     ) : (
       <EmptyState>
-        <h1>Ingen aktiviteter i feeden</h1>
+        <h2 className={styles.emptyState}>Ingen aktiviteter i feeden</h2>
```

Comment:
If you're overriding pretty much every style from `<h2>` in `styles.emptyState` anyway, I guess you might as well make this a `<div>` or a `<p>` or something? I agree with the change btw, looks less intrusive, well done!

Reply:
Actually not sure what would be the best choice, as <h2> has a semantic meaning, but as it's not actually a title or important text, ,so maybe a <p> would be better.